messenger/make.py

```diff
@@ -67,9 +76,55 @@ def build_octave():
     do_build(make_cmd, 'messenger.mex')
 
 
+def which_matlab():
```

Comment:
This will all be replaced by the pexpect stuff that @blink1073. Does that work well on Windows?